Initium-Odp/war/odp/javascript/script.js

```diff
@@ -1009,21 +1008,24 @@ function reloadPagePopup(quietly)
 {
 	if (currentPopupStackIndex==0)
 		return;
-	
+
 	var pagePopupId = "page-popup"+currentPopupStackIndex;
 	var content = $("#"+pagePopupId+"-content");
-	
+	var iframe = content.find("#"+pagePopupId+"-iframe");
+	if (iframe!=null)
+		content = iframe;
+
 	var url = content.attr("src");
+
+	if (quietly)
```

Comment:
Actually im not sure
On 2016-04-14 11:33 AM, "Nikolas Gauvreau" <nikolasarmstrong@gmail.com>
wrote:

> No probably not....
> On 2016-04-14 11:02 AM, "Anton Alenov" <notifications@github.com> wrote:
>
>> In Initium-Odp/war/odp/javascript/script.js
>> <https://github.com/Emperorlou/initium-odp/pull/7#discussion_r59733318>:
>>
>> >  	var url = content.attr("src");
>> > +
>> > +	if (quietly)
>>
>> Actually, is it even possible to reload an iframe in this way?
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/Emperorlou/initium-odp/pull/7/files/1e262965a144e3c05dcf3c3e72c92e0b4e824520#r59733318>
>>
>